Products/ZenUI3/browser/resources/js/zenoss/HierarchyTreePanel.js

```diff
@@ -729,8 +729,16 @@
             // is the top level organizer (e.g. Locations, Groups)
             return orgPieces[3] === targetPieces[3];
         },
-        refresh:function (callback, scope) {
-            this.getStore().load({
+        refresh:function (callback, scope, shift, direction) {
```

Comment:
This is too high level here since it is only for the network tree. This should be in the network tree subclass.